src/types/app.d.ts

```diff
@@ -26,3 +26,30 @@ interface IOwnerInfo {
   pictureUrl: string;
   state: string;
 }
+
+
+interface IDriverInformation {
+  // Personal Information
+  fullName: string;
+  dateOfBirth: string; // ISO string format when saved to cookie
+  gender: "Male" | "Female" | "Other";
```

Comment:
```suggestion
  gender: "Male" | "Female" |;
```
No LGBTQ things here abeg 😅